src/gc/gc.h

```diff
@@ -380,16 +380,12 @@ class GCHeap {
 
     static GCHeap *GetGCHeap()
     {
-#ifdef CLR_STANDALONE_BINDER
-        return NULL;
-#else
         LIMITED_METHOD_CONTRACT;
 
         _ASSERTE(g_pGCHeap != NULL);
         return g_pGCHeap;
-#endif
     }
-    
+
 #ifndef CLR_STANDALONE_BINDER
```

Comment:
It may be needed when the GC is building in ProjectN tree - let me double check. (We are maintaining exactly same GC source file across different repos/products.)